src/kind_lab_to_nwb/rat_behavioural_phenotyping_2025/requirements.txt

```diff
@@ -1,4 +1,5 @@
 ndx-events==0.2.1
+ndx-sound==0.2.1
 openpyxl
 neuroconv[video]
```

Comment:
Should also pin to main once https://github.com/catalystneuro/neuroconv/pull/1325 is merged